.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+# Description
+
+Fixes #[issue number]
+[Please include a summary of the changes and the related issue]
+
+## Demo
+
+[If possible, please include a screenshot or gif/video, it'll make it easier for reviewers to understand the scope of the changes and how the change is supposed to work. If you're introducing something new or changing the existing patterns, please share a Loom and explain what decisions you've made and under what circumstances]
+
+## Release notes
+
+[Include a human-readable description that could help to prepare a changelog]
```

Comment:
-1 I think the PR title should be that human readable line.

Reply:
good point

Reply:
added a checklist item, @Dschoordsch I would appreciate it if you could take a look